accounting/invoice/models.py

```diff
@@ -324,7 +327,7 @@ def get_folder_id(root, path=None):
             path=[str(self.billing_start_date.year), 'invoices-in', str(self.billing_start_date.month)]
         )
         file = drive.CreateFile({
-            'title': invoice_file.split('/')[-1],
+            'title': invoice_file.split('/')[-1].split('|')[-1],  # TODO: Make this cleaner.
```

Comment:
@UmanShahzad How about:

```python
    @staticmethod   
    def google_drive_file_name(invoice_file_path):
        """
        Return file name under which to store invoice that `invoice_file_path` points to on Google Drive.

        Assume that original file name contains a random prefix
        (format: '{uuid1}-{uuid2}|', where uuid-1 and uuid-2 are different UUID4s) to split off.
        """
        invoice_file_name = os.path.basename(invoice_file_path)
        return invoice_file_name.split("|")[-1]

    # ...

    def upload_to_google_drive(self, invoice_file_path): # NOQA
        # Note change to method signature ("invoice_file" parameter is now "invoice_file_path").
        # ...
        file = drive.CreateFile({
            'title': Invoice.google_drive_file_name(invoice_file_path)
            'parents': [{'id': month_folder}]
})

Reply:
Yeah that looks pretty good :+1: Thanks, will add it.